pandas/core/series.py

```diff
@@ -2734,6 +2736,15 @@ def from_csv(cls, path, sep=',', parse_dates=True, header=None,
 
         return result
 
+    def interpolate(self, *args, **kwargs):
+        if (is_datetime64_dtype(self) or
+                is_datetime64tz_dtype(self)) and self.isnull().any():
+            s2 = self.astype('i8').astype('f8')
+            s2[self.isnull()] = np.nan
+            return to_datetime(s2.interpolate(*args, **kwargs))
```

Comment:
I think you'll need to handle timezone information here. Once you do the `.astype('i8')` all TZ info is gone.

If someone has timezones, should it be `tz_convert('UTC') -> interploate -> tz_localize("UTC") -> tz_convert(original)`?

We'll want to test this around DST transitions...

Reply:
this should not be done here at all, rather in ``pandas.core.missing.py``